api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -121,11 +123,12 @@ public CohortDefinition getPatientsWithViralLoadSuppression() {
         "supp",
         EptsReportUtils.map(
             hivCohortQueries.getPatientsWithSuppressedViralLoadWithin12Months(), mappings));
+    cd.addSearch("baseCohort", EptsReportUtils.map(genericCohortQueries.getBaseCohort(), mappings));
     cd.addSearch(
         "onArtLongEnough",
         EptsReportUtils.map(
             getPatientsWhoAreMoreThan3MonthsOnArt(), "onDate=${endDate},location=${location}"));
-    cd.setCompositionString("supp AND onArtLongEnough");
+    cd.setCompositionString("supp AND baseCohort AND onArtLongEnough");
```

Comment:
Isn't `baseCohort` defined at `ReportDefinition` level,  does it need to be defined here also?

Reply:
removed baseCohort from the remain composition String